torch_lr_finder/lr_finder.py

```diff
@@ -460,6 +463,19 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None):
 
         # Plot loss as a function of the learning rate
         ax.plot(lrs, losses)
+
+        # Plot the suggested LR in a one-time loop
+        while suggestion:
+            try: 
+                mg = (np.gradient(np.array(losses))).argmin()
+            except:
+                print("Failed to compute the gradients, there might not be enough points.")
+                break
+
+            print(f"Min numerical gradient: {lrs[mg]:.2E}")
```

Comment:
This package supports Python 3.5 and above, f-strings were introduced in Python 3.6, you'll have to use `.format()` - this is why one of the CI jobs is failing. I would also propose changing the wording to something more user-friendly: `"Suggested LR {:.2E}".format(lrs[mg])`

Reply:
```suggestion
            print("Suggested LR: {:.2E}".format(lrs[mg]))
```